src/lexer.d

```diff
@@ -88,10 +89,10 @@ version (unittest)
 
 /***********************************************************
  */
-extern (C++) class Lexer
+class Lexer
```

Comment:
GDC's main entry function (d_parse_file) is in C++, and so will need to call into the static methods somehow.  ``Lexer::isValidIdentifier``, ``Lexer::idPool``